[2023-04-12T11:14:33.070+0000] {taskinstance.py:1080} INFO - Dependencies not met for <TaskInstance: data_extraction_dag.upload_positions_to_gcs manual__2023-04-12T11:14:16.454364+00:00 [skipped]>, dependency 'Task Instance State' FAILED: Task is in the 'skipped' state.
[2023-04-12T11:14:33.085+0000] {local_task_job.py:151} INFO - Task is not able to be run
[2023-04-12T11:14:42.376+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_extraction_dag.upload_positions_to_gcs manual__2023-04-12T11:14:16.454364+00:00 [queued]>
[2023-04-12T11:14:42.393+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_extraction_dag.upload_positions_to_gcs manual__2023-04-12T11:14:16.454364+00:00 [queued]>
[2023-04-12T11:14:42.394+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T11:14:42.395+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2023-04-12T11:14:42.396+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T11:14:42.421+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): upload_positions_to_gcs> on 2023-04-12 11:14:16.454364+00:00
[2023-04-12T11:14:42.435+0000] {standard_task_runner.py:55} INFO - Started process 1533 to run task
[2023-04-12T11:14:42.442+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_extraction_dag', 'upload_positions_to_gcs', 'manual__2023-04-12T11:14:16.454364+00:00', '--job-id', '899', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmptdhhwv5o']
[2023-04-12T11:14:42.451+0000] {standard_task_runner.py:83} INFO - Job 899: Subtask upload_positions_to_gcs
[2023-04-12T11:14:42.568+0000] {task_command.py:389} INFO - Running <TaskInstance: data_extraction_dag.upload_positions_to_gcs manual__2023-04-12T11:14:16.454364+00:00 [running]> on host a2e619923d04
[2023-04-12T11:14:42.661+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_extraction_dag
AIRFLOW_CTX_TASK_ID=upload_positions_to_gcs
AIRFLOW_CTX_EXECUTION_DATE=2023-04-12T11:14:16.454364+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-12T11:14:16.454364+00:00
[2023-04-12T11:14:42.663+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-12T11:14:42.687+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_extraction_dag, task_id=upload_positions_to_gcs, execution_date=20230412T111416, start_date=20230412T111442, end_date=20230412T111442
[2023-04-12T11:14:42.727+0000] {taskinstance.py:1080} INFO - Dependencies not met for <TaskInstance: data_extraction_dag.upload_positions_to_gcs manual__2023-04-12T11:14:16.454364+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2023-04-12T11:14:42.737+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-12T11:14:42.740+0000] {local_task_job.py:151} INFO - Task is not able to be run
[2023-04-12T11:14:42.755+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
